app/controllers/claims/pages_controller.rb

```diff
@@ -0,0 +1,2 @@
+class Claims::PagesController < ApplicationController
+end
```

Comment:
Don't we need an index controller method here?

Reply:
No. Rails will default to rendering the view if it finds one. :)